lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -28,7 +28,7 @@ def __init__(
     ):
         gw_ip_parsed = None
         try:
-            gw_ip_parsed = ipaddress.ip_address(gw_ip)
+            gw_ip_parsed = ipaddress.ip_address(str(gw_ip))
```

Comment:
Same here, but we actually catch the ValueError that happens if the IP is `None`. I think you can just add a `type: ignore`.